docs/source/upgrade_notes.rst

```diff
@@ -2,6 +2,10 @@
 
 Upgrade Notes
 =============
+|st2| v1.6.1
+------------
+
+* st2ctl reload now also reloads all the actions. Prior to this release we used to register actions, aliases, sensors, triggers and configs. We now also register sensors.
```

Comment:
@samirsss This reads a bit weird.

I assume it should say "st2ctl reload now also registers all the rules by default". And "Now rules are also registered by default".